src/test/integration/controllers/accounts.controller.e2e-spec.ts

```diff
@@ -0,0 +1,252 @@
+                        address: "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqhllllsajxzat",
+                        balance: expect.any(String),
+                        nonce: 0,
+                        timestamp: expect.any(Number),
+                        shard: 4294967295,
+                        ownerAddress: "erd1yj4jlay9rrzahran7jxk89gsg9frxw6l5qyca9dqhp8c4f5e0vdsytwkvl",
+                    },
+                ];
+
+            await request(app.getHttpServer())
+                .get(`${path}?${params}`)
+                .expect(200)
+                .then(res => {
+                    expect(res.body).toStrictEqual(expected);
+                });
+        });
+
+        it('should return 400 Bad Request if owner address is not correct', async () => {
+            const params = new URLSearchParams({
+                'ownerAddress': 'erd1yj4jlay9rrzahgsg9frxw6l5qyca9dqhp8c4f5e0vdsytwkvl',
+            });
+            await request(app.getHttpServer())
+                .get(`${path}?${params}`)
+                .expect(400)
+                .then(res => {
+                    expect(res.body.message).toEqual("Validation failed for argument 'ownerAddress' (a bech32 address is expected)");
+                });
+        });
+
+        [
+            {
+                filter: 'sort',
+                value: 'balance',
+            },
+        ].forEach(({ filter, value }) => {
+            describe(`when filter ${filter} is applied`, () => {
+                it(`should return accounts details based on ${filter} with value ${value} and ordered descendent `, async () => {
+                    const order = 'desc';
+
+                    await request(app.getHttpServer())
+                        .get(`${path}?${filter}=${value}&${order}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(res.body).toBeDefined();
+                            expect(Number(res.body[0].balance)).toBeGreaterThan(Number(res.body[1].balance));
+                        });
+                });
+            });
+        });
+
+        [
+            {
+                filter: 'sort',
+                value: 'timestamp',
+            },
+        ].forEach(({ filter, value }) => {
+            describe(`when filter ${filter} is applied`, () => {
+                it(`should return accounts details based on ${filter} with value ${value} and ordered descendent `, async () => {
+                    const order = 'desc';
+
+                    await request(app.getHttpServer())
+                        .get(`${path}?${filter}=${value}&${order}`)
+                        .expect(200)
+                        .then(res => {
+                            expect(res.body).toBeDefined();
+                            expect(res.body[0].timestamp).toBeGreaterThanOrEqual(res.body[1].timestamp);
+                        });
+                });
+            });
+        });
+
+        test.each`
```

Comment:
This test is good, but I consider that you can remove the following test cases because are the same:

- should return 25 accounts available on blockchain
- should return 1 account details
- should return 7 accounts